docs/package.json

```diff
@@ -68,6 +68,7 @@
     "react-helmet": "^5.2.0",
     "react-router": "^4.3.0",
     "react-router-dom": "^4.3.1",
+    "react-router-hash-link": "^1.2.1",
```

Comment:
Do we still need to add this new module? I believe we ended up removing all the hash links.

Reply:
Ah, after the sidenav reorg, we do not. I'll remove.